server/vulnerabilities/msrc/xml/product.go

```diff
@@ -0,0 +1,43 @@
+package xml
+
+import "strings"
+
+// XML elements related to the 'prod' namespace used to describe Microsoft products
```

Comment:
this general comment helps, but I'd be nice to have comments for each of the structs below

Reply:
Thanks, I'll add references to the XML schema definition. Besides that I don't what else to add without being redundant, from my point of view the type names are self explanatory, e.g. a `Product` is used for defining, welp a product.

Reply:
gotcha, I was thinking of a brief explanation mainly for things like `ProductBranch`, but you have more context than me, if you think it can be properly inferred then we're fine.